src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java

```diff
@@ -31,12 +31,14 @@
             MODULE, DATE, SESSION_TYPE
     );
 
-    public static final String MESSAGE_FILTERED_SESSIONS_SUCCESS = "Filtered Session List: %s";
```

Comment:
Why did you change the success message tho? i thought it was nice that the user gets to see what filters have just been applied

Reply:
Yup it does! Now instead it shows only the ones that have been specified.
Before it did this: Date | Module | Type

Now its
Date:
Module:
Type:

Reply:
Can test then will see!